### PR TITLE
Added/fixed, dogfacts.js without module and catfacts.js 

### PR DESCRIPTION
**Pull request by MrJacz**
dogfacts.js without module because the module was just randomizing an array.
The catfacts.js old endpoint must have been broken or down tho this API/endpoint has been around for a while and works fine.
This is from #83 because of me deleting the fork and not being able to fix linter issues 😢 

~Jacz

### CODE REVIEW

File: commands/Fun/dogfacts.js [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+  "Unlike cats, dogs are not exclusively carnivores: they can adapt to different foods, and a percentage of their diets can be fruits and vegetables.",
+  "Female wolves have been known to travel great distances to regurgitate full meals for their hungry pups.",
+  "There are hundreds of breeds of dogs.",
+  "In Egypt, a person bitten by a rabid dog was encouraged to eat the roasted liver of a dog infected with rabies to avoid contracting the disease. The tooth of a dog infected with rabies would also be put in a band tied to the arm of the person bitten. The menstrual blood of a female dog was used for hair removal, while dog genitals were used for preventing the whitening of hair.",
+  "Dogs can get jealous when their humans display affection toward someone or something else.",
+  "Weird dog laws include allowing police offers in Palding, Ohio, to bite a dog to quiet it. In Ventura County, California, cats and dogs are not allowed to have sex without a permit.",
+  "The largest dog was an English Mastiff who weighed 343 pounds.",
+];
+exports.run = (client, msg) => {
+  msg.channel.send(facts[Math.floor(Math.random() * facts.length)]);
```
Comment (kyranet) [inline commands/Fun/dogfacts.js]: Make this function return with `return` so Komada can handle the promise.

File: commands/Fun/dogfacts.js [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+};
+
+exports.help = {
```
Comment (kyranet) [inline commands/Fun/dogfacts.js]: To bring consistency with the rest of pieces, please put `exports.conf` above `exports.help`

File: commands/Fun/catfacts.js [hunk 3]
```diff
@@ -1,8 +1,7 @@
-const request = require("snekfetch");
+const snekfetch = require("snekfetch");
 
 exports.run = async (client, msg) => {
-  const { facts } = await request.get("http://catfacts-api.appspot.com/api/facts").then(data => JSON.parse(data.text));
-  return msg.channel.send(`📢 **Catfact:** *${facts[0]}*`);
+  snekfetch.get("https://catfact.ninja/fact").then(res => msg.channel.send(`📢 **Catfact:** *${res.body.fact}*`));
```
Comment (kyranet) [inline commands/Fun/catfacts.js]: Same here, return it.

Comment (MrJacz) [reply commands/Fun/catfacts.js]: Okay boss

**Comment by ghost** [comment]
@kyranet Done 👯‍♂️

**Comment by ghost** [comment]
Travis CI arent u werid

**Review by MrJacz — COMMENTED** [review]
🔥

**Pull request merged**